UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -122,4 +138,9 @@ public void CancelJump()
     {
         isJumping = false; //This will stop the reduction to the gravity, which will then quickly pull down the character
     }
+
+    private void OnControllerColliderHit(ControllerColliderHit hit)
+    {
+        hitNormal = hit.normal;
```

Comment:
Why dont you assign `canJump` here? Instead now the same limit is being calculated every frame, instead of every touch. The latter is more performant!